app/javascript/components/concept-exercise-graph/Concept.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { CompleteIcon } from './CompleteIcon'
+
+import { IConcept, ConceptState } from './concept-types'
+
+export const Concept = ({
+  index,
+  slug,
+  conceptExercise,
+  uuidOfExercise,
+  status,
+  handleEnter,
+  handleLeave,
+  isActive,
+  isDimmed,
+  adjacentConcepts,
+}: IConcept & {
+  isActive: boolean
+  isDimmed: boolean
+  adjacentConcepts: string[]
+}) => {
+  const name = slugToTitlecase(slug)
+
+  let classes = 'card'
+  classes += ` card-${status}`
+  classes += isActive ? ' card-active' : ''
+  classes += isDimmed ? ' dim' : ''
+  adjacentConcepts.forEach((adjacentConcept: string): void => {
+    classes += ` adjacent-to-${adjacentConcept}`
+  })
+
+  return (
+    <section
```

Comment:
The card should be an `<a`, and it's `href` should be driven by a field in the JSON called `web_url`, which each concept will have.

There might be some concepts that don't have `web_url` (locked ones), so we might need to pivot on this as to whether an `<a>` or something else (`<section>` still maybe?) is rendered. We've not decided this though, so for now you could just use `<a>` everywhere.